<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameObjectAudioSourceComponentWidgetQt</class>
 <widget class="QWidget" name="GameObjectAudioSourceComponentWidgetQt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>326</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>356</width>
    <height>326</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Game Object Audio Source Components</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow, QWidget, QDialog, QMenuBar::item
{
	background-color: rgb(50, 50, 50);
	color: rgb(255, 255, 255);
}

QHeaderView::section
{
	background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:0, y2:0.5, stop:0 rgba(50, 50, 50, 255), stop:1 rgba(75, 75, 75, 255));
	padding-left: 4px;
	border: 1px solid rgb(110, 110, 110);
}

QMenuBar::item:selected, QMenu::item:selected
{
    background-color: rgb(120, 120, 120);
	border: 1px solid rgb(200, 200, 200);
}

QPushButton:hover
{
    background-color: rgb(75, 75, 75);
}

QPushButton:pressed
{
    background-color: rgb(100, 100, 100);
}

QLineEdit
{
	border-width: 2px;
	border-style: inset;
	border-color: rgb(60, 60, 60);
	border-radius: 5px;
	padding: 1px;
}

QLineEdit:focus
{
	background-color: rgb(75, 75, 75);
}

QDockWidget::title
{
	background-color: rgb(65, 65, 65);
	padding: 5px;
}

QTabBar::tab 
{
	background-color: qlineargradient(spread:pad, x1:1, y1:0.5, x2:0, y2:0.5, stop:0 rgba(60, 60, 60, 255), stop:0.5 rgba(80, 80, 80, 255), stop:1 rgba(60, 60, 60, 255));
	padding: 5px;
	borde-width: 3px; 
	border-color: rgb(110, 110, 110);
	border-style: inset;
	border-radius: 8px;
	font-size: 11px;
}

QTabBar::tab:selected 
{
	background-color: qlineargradient(spread:pad, x1:1, y1:0.5, x2:0, y2:0.5, stop:0 rgba(60, 60, 60, 255), stop:0.5 rgba(120, 120, 120, 255), stop:1 rgba(60, 60, 60, 255));
	border-style: outset;
    border-bottom-color: rgb(60, 60, 60);
}

QTreeWidget
{
	alternate-background-color: rgb(60, 60, 60);
}

QTreeWidget::branch:has-children
{
	image: url(:/XEngine_Editor/Resources/Images/branch-closegrey.png) 0;
}

QTreeWidget::branch:has-children:open
{
	image: url(:/XEngine_Editor/Resources/Images/branch-opengrey.png) 0;
}

QTreeWidget::branch:has-children:open:selected
{
	image: url(:/XEngine_Editor/Resources/Images/branch-openblue.png) 0;
}

QTreeWidget::branch:has-children:!open:selected
{
	image: url(:/XEngine_Editor/Resources/Images/branch-closeblue.png) 0;
}
</string>
  </property>
  <widget class="QTreeWidget" name="m_AudioSourceTree">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="autoScrollMargin">
    <number>0</number>
   </property>
   <property name="alternatingRowColors">
    <bool>false</bool>
   </property>
   <property name="indentation">
    <number>0</number>
   </property>
   <property name="rootIsDecorated">
    <bool>true</bool>
   </property>
   <property name="itemsExpandable">
    <bool>false</bool>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <column>
    <property name="text">
     <string>Audio Sources</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="m_AddAudioSource">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Audio Source</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_RemoveAudioSource">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Audio Source</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_AudioAssetTextBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>161</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Script Asset</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="m_AudioAssetLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>162</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Asset:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_SearchAudioAsset">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>159</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_LoopCheckBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>290</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Loop:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_Is3DCheckBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>290</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Is 3D:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="m_VolumeSB">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>199</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="m_PitchSB">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>257</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-10000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="m_MinimumDistanceSB">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>229</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>1.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="m_AttenuationSB">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>287</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_NameAudioSourceTextBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Script Asset</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="m_AudioName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_VolumeLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>201</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Volume:</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_AttenuationLabel">
   <property name="geometry">
    <rect>
     <x>37</x>
     <y>289</y>
     <width>63</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Attenuation:</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_PitchLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>31</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Pitch:</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_MinDistanceLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Minimum Distance:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="m_SoundMusicGB">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Behave as</string>
   </property>
   <widget class="QRadioButton" name="m_IsMusic">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Music</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="m_IsSound">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Sound</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="m_ChannelCountLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Channel Count:</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_SampleRateLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sample Rate:</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_DurationLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Duration:</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_SampleRateLabelText">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>230</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>----</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_DurationLabelText">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>----</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_ChannelCountLabelText">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>----</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
